app/lib/services/voip/MediaCallEvents.ios.test.ts

```
@@ -228,6 +237,22 @@ describe('setupMediaCallEvents — VoipPushTokenRegistered (iOS)', () => {
+	it('calls debug() with the token but NOT log() when VoipPushTokenRegistered is fired', () => {
+		setupMediaCallEvents(makeTestAdapters());
+		emitNativeVoipEvent('VoipPushTokenRegistered', { token: 'voip-token-sensitive' });
+
+		// Access the mock functions that were passed to the MediaCallLogger class
+		const { __mockDebug, __mockLog } = jest.requireMock('./MediaCallLogger');
+
+		// debug() must be called (sensitive data goes through debug level)
+		expect(__mockDebug).toHaveBeenCalledWith(
+			expect.stringContaining('Registered VoIP push token:'),
+			'voip-token-sensitive'
+		);
```

ESLint

[error] Replace `⏎↹↹↹expect.stringContaining('Registered·VoIP·push·token:'),⏎↹↹↹'voip-token-sensitive'⏎↹↹` with `expect.stringContaining('Registered·VoIP·push·token:'),·'voip-token-sensitive'`

(prettier/prettier)